resources/tax_type/ca_ab_gst.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Canada - Alberta",
+    "generic_label": "GST",
+    "display_inclusive": false,
```

Comment:
Cool, I'd submit a pull request with your changes but it makes a new PR and would prefer if I could help on yours magically. (new to contrib on github)